scp/scpHandlingEventThatInitiatesAgentSCPProgram.cpp

```diff
@@ -13,6 +13,19 @@
 
 namespace scp
 {
+
+namespace
+{
+class SCPResult : public ScResult
+{
+public:
+  SCPResult(sc_result code)
+    : ScResult(code)
+  {
+  }
+};
```

Comment:
move this class to separate cpp/hpp files

Reply:
This is not a valid reason
Most agents are only used in module file but they are placed in their own files

Reply:
This is not java, in which all classes must have their own file. In C++ there is such a concept as encapsulation by hiding in a file. It is unlikely that SCPResult class will be used anywhere else